README.md

```diff
@@ -4,18 +4,21 @@ This is a [Viam module](https://docs.viam.com/manage/configuration/#modules) for
 
 ## Getting Started
```

Comment:
This isn't a comment for the README, but there's nowhere else to put it.

I noticed this is v0.0.2. Normally, unversioned software starts at v0.1.0. See https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase. That's a good doc to read if you want to understand semantic versioning better.

Reply:
Okay, I can make the "first" version the one I cut today then